src/todolist.js

```diff
@@ -0,0 +1,87 @@
+import DeleteIcon from './delete_black_24dp.svg';
+
+const toDoContainer = document.getElementById('todo-container');
+export default class ToDoList {
+  createEmpty = () => {
+    const todoes = [];
+    localStorage.setItem('todoes', JSON.stringify(todoes));
```

Comment:
- [ ] You are yet to implement the edit functionality as requested by the previous reviewer. Kindly fix that.

  The task description can be 'edited' but it does not persist in the localStorage after `blur` (leaving the task), on pressing the enter. Hence when I reload the task still remains the same.

  Tasks are added | Task is "edited" | Page is reloaded
  :---:|:---:|:---:
  ![dango-1](https://user-images.githubusercontent.com/36542986/160038740-623d83fa-ab39-4577-ac7a-f8c28d3b2327.jpg) | ![dango-2](https://user-images.githubusercontent.com/36542986/160038741-069ef775-5c45-4e97-83f0-cf1a959ad1ba.jpg) | ![dango-1](https://user-images.githubusercontent.com/36542986/160038740-623d83fa-ab39-4577-ac7a-f8c28d3b2327.jpg)